Proposition for CA question

laumane opened the pull request:
I tried to sum up as much as I can and detailed a bit X.509 standard.

laumane committed:
Proposition for CA question

bregman-arie commented on exercises/security/README.md:
```diff
@@ -16,6 +16,18 @@
 <details>
 <summary>What is a Certificate Authority?</summary><br><b>
+ 
+ A certificate Authority certifies the authenticity of the public key delivered by the website. It prevents [man-in-the-middle](https://en.wikipedia.org/wiki/Man-in-the-middle_attack) attacks by providing a lot of information which identifie the public key. Importante information provided inside a [X.509](https://www.ssl.com/faqs/what-is-an-x-509-certificate/) certificate are like :
```
Certificate Authority doesn't have to be related to websites. It's more generic than the specific implementation mentioned here.

laumane committed:
Update CA question

I would like to change a little bit the question to include the meaning of a certification. Does it mind you ?
